sys/netbsd/socket.txt

```diff
@@ -42,7 +46,7 @@ socket_type = SOCK_STREAM, SOCK_DGRAM, SOCK_RAW, SOCK_RDM, SOCK_SEQPACKET, SOCK_
 accept_flags = SOCK_NONBLOCK, SOCK_CLOEXEC, SOCK_NOSIGPIPE
 shutdown_flags = SHUT_RD, SHUT_WR, SHUT_RDWR
 send_flags = MSG_OOB, MSG_PEEK, MSG_DONTROUTE, MSG_EOR, MSG_NOSIGNAL
-recv_flags = MSG_CMSG_CLOEXEC, MSG_OOB, MSG_PEEK, MSG_WAITALL
+recv_flags = MSG_CMSG_CLOEXEC, MSG_OOB, MSG_PEEK, MSG_DONTWAIT, MSG_WAITFORONE
```

Comment:
Personally, I would list all the flags from https://nxr.netbsd.org/xref/src/sys/sys/socket.h#489

Even if some are unused. recv() should use MSG_WAITALL but it is removed here? It does not cost much to just list all possible flags even if certain syscalls don't handle all of them.